packages/atomic/src/components/facets-v1/atomic-numeric-facet/atomic-numeric-facet.tsx

```diff
@@ -110,16 +115,14 @@ export class AtomicNumericFacet
   @Prop() public field!: string;
   /**
    * The number of values to request for this facet, when there are no manual ranges.
+   * If the number of values is 0, no ranges will be displayed.
    */
   @Prop() public numberOfValues = 8;
   /**
    * Whether this facet should contain an input allowing users to set custom ranges.
+   * Depending on the field, the input can allow either decimal or integer values.
    */
-  @Prop() public withInput = false;
-  /**
-   * Whether this facet should contain a defined list of values.
-   */
-  @Prop() public withRanges = true;
+  @Prop() public withInput?: NumberInputType;
```

Comment:
Should this option be required since it depends on the configured `field`?

Reply:
When undefined, we don't show the input 

> Whether this facet should contain an input allowing users to set custom ranges.